src/ostorlab/runtimes/lite_local/agent_runtime.py

```diff
@@ -0,0 +1,263 @@
+"""Runtime agent module gives access to methods to create the specific agent configurations & run the agent.
```

Comment:
I think you can delete this one and simply use the one from localruntime for the moment.